doc/source/quickstart/quickstart-Hologres.md

```diff
@@ -0,0 +1,998 @@
+- Hologres Initialization
+- Data Uploading
+- Query Time
+- Commands in Linux and SQL 
+
+## Hologres Introduction
+
+Hologres seamlessly opens up with the big data ecology, supporting high concurrency and low delay analysis and processing of PB-level data, allowing you to easily and economically use existing BI tools for multi-dimensional analysis, perspective and business exploration of data.
+
+For now, we have one stand-alone version of Hologres to try!
+
+## Data Preparation
+
+1.Data Source
+
+First,download a large amount of data to get familiar with Hologres. In this blog,we take some flight data from [the USA government example](http://www.transtats.bts.gov/DL_SelectFields.asp?Table_ID=236&DB_Short_Name=On-Time) without a license and therefore assumed to be in the public domain in the United States.
+
+Github has a repro containing scripts and loaders for ["On-Time Performance"](https://github.com/Percona-Lab/ontime-airline-performance) data.
+
+2.Download the Dataset
+
+If you are in Linux, go with the following script and download the data；
+One thing need to remember, choose the right directory before you start.
+
+```
+for s in `seq 1987 2018`
+do
+for m in `seq 1 12`
+do
+wget https://transtats.bts.gov/PREZIP/On_Time_Reporting_Carrier_On_Time_Performance_1987_present_${s}_${m}.zip
+done
+done
+```
+
+![image](../images/Quickstart/quickstart-Hologres/datadownload1.gif)
+
+![image](../images/Quickstart/quickstart-Hologres/datadownload2.gif)
+
+Data is stored in CSV (with , delimiter) format.  Total uncompressed CSV files size is about 79852548 Bytes (about 77GB). 
+Note: in 1987, only 10,11,12 (3 months) data is available for download.
+Note: the following 14 files fails to upload in Hologres for the invalid byte sequence for encoding.
+
+| NO | FILES |
+|---|---|
+| 1 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_1.csv |
+| 2 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_2.csv |
+| 3 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_3.csv |
+| 4 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_4.csv |
+| 5 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_5.csv |
+| 6 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_6.csv |
+| 7 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_7.csv |
+| 8 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_8.csv |
+| 9 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_9.csv |
+| 10 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_10.csv |
+| 11 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_11.csv |
+| 12 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2001_12.csv |
+| 13 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2002_1.csv |
+| 14 | On_Time_Reporting_Carrier_On_Time_Performance_(1987_present)_2002_2.csv |
+
+
+we better delete these 14 files by:
```

Comment:
~~we better delete these 14 files by:~~ Let's delete the 14 files via following commands